play-ws-standalone/src/main/scala/play/api/libs/ws/StandaloneWSRequest.scala

```diff
@@ -162,7 +162,8 @@ trait StandaloneWSRequest {
   }
 
   /**
-   * Returns this request with the given query string parameters, discarding the existing ones.
+   * Returns this request with the given query string parameters, discarding the existing ones for this request.
+   * But when `play.ws.ahc.useCookieStore` is true, there is a cookie store that is global and keeps cookies between requests.
    *
```

Comment:
I have a similar issue here.

I think the following will be more clear:

```
Returns this request with the given cookies, discarding the existing ones. In general, cookies set on a request are used only on that request and then discarded. When `play.ws.ahc.useCookieStore` is enabled, there will be a global cookie store that keeps cookies between requests.
```

But I don't know if it is correct.

Reply:
It is clearer and correct. I will use it in both Scala and Java docs.